[2022-01-25 09:51:49,036] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.load_data 2022-01-25T09:46:10.257136+00:00 [queued]>
[2022-01-25 09:51:49,071] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.load_data 2022-01-25T09:46:10.257136+00:00 [queued]>
[2022-01-25 09:51:49,071] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 09:51:49,072] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-01-25 09:51:49,072] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 09:51:49,088] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): load_data> on 2022-01-25T09:46:10.257136+00:00
[2022-01-25 09:51:49,095] {standard_task_runner.py:52} INFO - Started process 43 to run task
[2022-01-25 09:51:49,103] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'classification', 'load_data', '2022-01-25T09:46:10.257136+00:00', '--job-id', '31', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/classification.py', '--cfg-path', '/tmp/tmpk39eu37i', '--error-file', '/tmp/tmpbhzyy4di']
[2022-01-25 09:51:49,109] {standard_task_runner.py:77} INFO - Job 31: Subtask load_data
[2022-01-25 09:51:49,177] {logging_mixin.py:104} INFO - Running <TaskInstance: classification.load_data 2022-01-25T09:46:10.257136+00:00 [running]> on host 2dc98551bc76
[2022-01-25 09:51:49,230] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=classification
AIRFLOW_CTX_TASK_ID=load_data
AIRFLOW_CTX_EXECUTION_DATE=2022-01-25T09:46:10.257136+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-25T09:46:10.257136+00:00
[2022-01-25 09:52:40,929] {python.py:118} INFO - Done. Returned value was: None
[2022-01-25 09:52:40,978] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=classification, task_id=load_data, execution_date=20220125T094610, start_date=20220125T095149, end_date=20220125T095240
[2022-01-25 09:52:41,113] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-01-25 09:52:41,178] {local_task_job.py:146} INFO - Task exited with return code 0
